integration-tests/gradle/src/test/java/io/quarkus/gradle/AddExtensionToSingleModuleProjectTest.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.gradle;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+
+public class AddExtensionToSingleModuleProjectTest extends QuarkusGradleWrapperTestBase {
+
+    private static final int DEFAULT_BUFFER_SIZE = 1024 * 4;
+
+    @Test
+    public void testAddAndRemoveExtension() throws IOException, URISyntaxException, InterruptedException {
+
+        final File projectDir = getProjectDir("add-remove-extension-single-module");
+
+        runGradleWrapper(projectDir, ":addExtension", "--extensions=hibernate-orm");
+
+        final Path Build = projectDir.toPath().resolve("build.gradle");
+        assertThat(Build).exists();
+        assertThat(readFile(Build)).contains("implementation 'io.quarkus:quarkus-hibernate-orm'");
+
+        runGradleWrapper(projectDir, ":removeExtension", "--extensions=hibernate-orm");
+        assertThat(readFile(Build)).doesNotContain("implementation 'io.quarkus:quarkus-hibernate-orm'");
+
+    }
+
+    private static String readFile(Path file) throws IOException {
```

Comment:
You can use `Files.readAllBytes` instead

Reply:
Will do